### DISCUSSION TITLE
Any way to disable Accept-Ranges and 206 partial responses?

### DISCUSSION THREAD

**Discussion by asier-burns** [vercel/next.js#44920] [Help]
Because of problems with a CDN we want to force Accept-Ranges: none and never return partial content 206, always return 200 with full content even if the request includes a range, ignore it.

Is this possible?

**Comment by alfrescian** [comment]
Did you try remove the header in a middleware as shown above or did you find another solution?

```function noRangeRequestsForAsssetsHandler(req: NextRequest) {
  const { pathname } = req.nextUrl;

  const isMatchingPath = pathname.startsWith('/next/_static'); // e.g.
  if (isMatchingPath) {
    const requestHeaders = new Headers(req.headers);
    requestHeaders.delete('range');

    return NextResponse.next({
      request: {
        headers: requestHeaders,
      },
    });
  }
}```

**Reply by funsjanssen** [reply]
@alfrescian This does indeed work. There is a minor error in this example code. Spent an hour trying to figure out why it didn't work 😅 

`const isMatchingPath = pathname.startsWith('/next/_static'); // e.g.
`

should be

`const isMatchingPath = pathname.startsWith('/_next/static'); // e.g.
`
